src/harness/sas_connector.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -- encoding: utf-8 --
+
+__author__ = "Francisco Benavides (francisco.benavides@federatedwireless.com)"
```

Comment:
We might not want to put author information here since other people might also work on this file.

We might need to put apache license text in every file.